README.md

```diff
@@ -85,23 +86,40 @@ docker build --build-arg TF_VERSION=1.15.0-gpu -t $USER/deepcell-tf .
 ```bash
 # NV_GPU refers to the specific GPU to run DeepCell on, and is not required
 
-# Mounting the codebase, scripts and data to the container is also optional
-# but can be handy for local development
-
 NV_GPU='0' nvidia-docker run -it \
   -p 8888:8888 \
   $USER/deepcell-tf:latest
 ```
 
 It can also be helpful to mount the local copy of the repository and the scripts to speed up local development.
 
+However, if you are going to mount a local version of the repository, you must first run the docker image without the local repository mounted so that the c extensions can be compiled and then copied over to your local version.
+
 ```bash
+# First run the docker image without mounting externally
+NV_GPU='0' nvidia-docker run -it \
+  -p 8888:8888 \
+  $USER/deepcell-tf:latest
+  
+# Use ctrl-p, ctrl-q to exit the running docker image without shutting it down
+
+# Then, get the container_id corresponding to the running image of deepcell
+container_id=$(docker ps -q --filter ancestor="$USER/deepcell-tf") 
+
+# Copy the compiled c extensions into your local version of the codebase:
+docker cp "$CONTAINER_ID:/usr/local/lib/python3.6/dist-packages/deepcell/utils/compute_overlap.cpython-36m-x86_64-linux-gnu.so" deepcell/utils/compute_overlap.cpython-36m-x86_64-linux-gnu.so
```

Comment:
Does this actually run?  I would think `$CONTAINER_ID` and `container_id` do not match.

Reply:
doh, I switched between the two when testing it locally and writing up the readme